specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/confidentialledger.json

```diff
@@ -0,0 +1,849 @@
+{
```

Comment:
You would still require breaking changes review, as one can not introduce breaking changes from last stable API version also. Imagine, Azure customers moving from stable version to newer preview version and seeing breaking changes.

Please reach out to breaking changes review board <azbreakchangereview@microsoft.com> for getting your changes reviewed.

Reply:
Yes going through the breaking change review process at the moment.